src/content/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration.mdx

```diff
@@ -21,314 +21,230 @@ After setting up our SonarQube integration, we give you a dashboard for your Son
+
+  ```yml
+  collect:
+    - domain: SonarQube
+      event_type: JVMSampleSonarQubeComputeEngineMetrics
+      beans:
+        - query: name=ComputeEngineDatabaseConnection
+          attributes:
+            - PoolInitialSize
+            - PoolActiveConnections
+            - PoolMaxActiveConnections
+            - PoolIdleConnections
+            - PoolMaxIdleConnections
+            - PoolMinIdleConnections
+            - PoolMaxWaitMillis
+            - PoolRemoveAbandoned
+            - PoolRemoveAbandonedTimeoutSeconds
+        - query: name=ComputeEngineTasks
+          attributes:
+            - PendingCount
+            - LongestTimePending
+            - InProgressCount
+            - ErrorCount
+            - SuccessCount
+            - ProcessingTime
+            - WorkerMaxCount
+            - WorkerCount
+            - WorkerUuids
+            - EnabledWorkerUuids
+  ```
+
+5. Add the following code snippet to `/etc/newrelic-infra/integrations.d/jmx-sonarqube-compute-engine-config.yml`:
+  ```yml
+  integrations:
+    - name: nri-jmx
+      env:
+        COLLECTION_FILES: /etc/newrelic-infra/integrations.d/jmx-sonarqube-web-config.yml
+        JMX_HOST: <YOUR_HOST>
+        JMX_PASS: admin
+        JMX_PORT: 9010
+        JMX_USER: admin
+        CONNECTION_URL: service:jmx:rmi://<YOUR_IP>:9010/jndi/rmi://<YOUR_IP>:9010/jmxrmi
+        REMOTE_MONITORING: "true"
+      interval: 15s
+      labels:
+        env: staging
+  ```
+6. Add the following code snippet to `/etc/newrelic-infra/integrations.d/jmx-sonarqube-web-config.yml`:
+
+  ```yml
+  integrations:
+    - name: nri-jmx
+      env:
+        COLLECTION_FILES: /etc/newrelic-infra/integrations.d/jmx-sonarqube-compute-engine-config.yml
+        JMX_HOST: <YOUR_HOST>
+        JMX_PASS: admin
+        JMX_PORT: 9011
+        JMX_USER: admin
+        CONNECTION_URL: service:jmx:rmi://<YOUR_IP>:9011/jndi/rmi://<YOUR_IP>:9011/jmxrmi
+        REMOTE_MONITORING: "true"
+      interval: 15s
+      labels:
+        env: staging
+  ```
+    </Step>  
+    <Step>
 ## Forward SonarQube logs to New Relic
 
-You can use our [log forwarding](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/) to forward SonarQube logs to New Relic.
-
-On Linux machines, your log file named `logging.yml` should be present in this path:
-
-```shell
-cd /etc/newrelic-infra/logging.d/
-```
-After creating the log file, add the following script to the `logging.yml` file:
-
-```yml
-logs:
-  - name: sonar_logs
-    file: /opt/sonarqube/logs/sonar.log
-    attributes: 
-      logtype: sonar_logs
-  - name: ce_logs
-    file: /opt/sonarqube/logs/ce.log
-    attributes:
-      logtype: sonar_ce_logs
-  - name: es_logs
-    file: /opt/sonarqube/logs/es.log
-    attributes: 
-      logtype: sonar_es_logs
-  - name: web_logs
-    file: /opt/sonarqube/logs/web.log
-    attributes:
-      logtype: sonar_web_logs
-``` 
-
+Follow these steps to forward SonarQube logs to New Relic:
```

Comment:
What level of detail do customers get by forwarding logs? Can they opt out of this if they have ingest concerns? In our other installs, logs tend to be optional but I noticed we have it as a required step here. We might want to explain how the ~fidelity of data gets degraded if they opt out (or maybe spin it positively to explain that logs are a key part oof the dashboard experience or w/e the reason is)

Reply:
here + in the last OHI i reviewed